docker/ubuntu1604/Dockerfile

```diff
@@ -71,9 +72,16 @@ RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-3.8 100 &&
     update-alternatives --install /usr/bin/scan-build scan-build /usr/bin/scan-build-3.8 100 && \
     update-alternatives --install /usr/bin/llvm-symbolizer llvm-symbolizer /usr/bin/llvm-symbolizer-3.8 100
 
+# Redis cannot start as a daemon on a Docker build.
+RUN sed -i "s/daemonize yes/daemonize no/g" /etc/redis/redis.conf
+
+# To preserve the logs for redis
+RUN chown -R redis:redis /var/log/redis/
+
 ADD supervisord.conf /etc/supervisor/supervisord.conf
 ADD docker-entrypoint.sh /sbin/docker-entrypoint.sh
 
-ENTRYPOINT ["/sbin/docker-entrypoint.sh"]
+#ENTRYPOINT ["/sbin/docker-entrypoint.sh"]
```

Comment:
Again this is needed for the internal build system, so it needs to be kept in here.